src/github.com/getlantern/flashlight/genconfig/README.md

```diff
@@ -1,10 +1,53 @@
-# Updating the global configuration
-
-1. Make sure you have REDISCLOUD_PRODUCTION_URL set as an environment variable -- see https://github.com/getlantern/too-many-secrets/blob/master/lantern_aws/config_server.yaml#L2
-1. If you want to embed custom chained servers, make sure to have them populated in fallbacks.yaml and generated using the private lantern_aws/etc/fetchcfg.py. You can do this as follows:
-```
-./fetchcfg.py vltok1 > fallbacks.yaml
-./fetchcfg.py >> fallbacks.yaml
-```
-1. Run ```./genconfig.bash```
-1. Run ```./cfg2redis.py --global cloud.yaml -```
+# genconfig
+
+**Generate the YAML file and embedded Go datastructures containing the network configuration for the Lantern client**
+
+
+## How it works
+
+Config generation currently follows this approach:
+
+1. Build a *model*, a complete list of working chained servers, masquerades and proxied sites.
+
+1. Fill the Go code templates (*.go), based on the model.
+
+1. Fill the YAML template, based on the model. The YAML template already has filled in the LFS round-robin (flashlight servers for domain fronting).
+
+## Using genconfig
+
+### Managing masquerade hosts
+
+The file domains.txt contains the list of masquerade hosts we use, and
+blacklist.txt contains a list of blacklisted domains that we exclude even if
+present in domains.txt.
+
+To alter the list of domains or blacklist:
+
+1. Edit [`domains.txt`](domains.txt) and/or [`blacklist.txt`](blacklist.txt)
+2. `go run genconfig.go -domains domains.txt -blacklist blacklist.txt`.
```

Comment:
IIRC there's a `genconfig.bash` which embodies latest recommended usage.  I think this file should point to that, because that's the one we're actually using, and it's more likely to be kept up to date than this README.